packages/tabs/test/lion-tabs.test.js

```diff
@@ -254,6 +256,92 @@ describe('<lion-tabs>', () => {
       tabs[0].dispatchEvent(new KeyboardEvent('keyup', { key: 'ArrowLeft' }));
       expect(el.selectedIndex).to.equal(2);
     });
+
+    it('selects next available not disabled tab if first tab is disabled', async () => {
+      const el = /** @type {LionTabs} */ (
+        await fixture(html`
+          <lion-tabs>
+            <button slot="tab" disabled>tab 1</button>
+            <div slot="panel">panel 1</div>
+            <button slot="tab">tab 2</button>
+            <div slot="panel">panel 2</div>
+            <button slot="tab">tab 3</button>
+            <div slot="panel">panel 3</div>
+          </lion-tabs>
+        `)
+      );
+      expect(el.selectedIndex).to.equal(1);
+    });
+
+    it('selects next available not disabled tab on [arrow-right] and [arrow-down]', async () => {
+      const el = /** @type {LionTabs} */ (
+        await fixture(html`
+          <lion-tabs>
+            <button slot="tab">tab 1</button>
+            <div slot="panel">panel 1</div>
+            <button slot="tab" disabled>tab 2</button>
+            <div slot="panel">panel 2</div>
+            <button slot="tab">tab 3</button>
+            <div slot="panel">panel 3</div>
+            <button slot="tab" disabled>tab 3</button>
+            <div slot="panel">panel 3</div>
+          </lion-tabs>
+        `)
+      );
+      const tabs = el.querySelectorAll('[slot=tab]');
+      tabs[0].dispatchEvent(new KeyboardEvent('keyup', { key: 'ArrowRight' }));
+      expect(el.selectedIndex).to.equal(2);
+    });
+
+    it('selects next available not disabled tab on [arrow-left] and [arrow-up]', async () => {
+      const el = /** @type {LionTabs} */ (
+        await fixture(html`
+          <lion-tabs>
+            <button slot="tab">tab 1</button>
+            <div slot="panel">panel 1</div>
+            <button slot="tab" disabled>tab 2</button>
+            <div slot="panel">panel 2</div>
+            <button slot="tab">tab 3</button>
+            <div slot="panel">panel 3</div>
+            <button slot="tab" disabled>tab 3</button>
+            <div slot="panel">panel 3</div>
+          </lion-tabs>
+        `)
+      );
+      el.selectedIndex = 2;
+      const tabs = el.querySelectorAll('[slot=tab]');
+      tabs[2].dispatchEvent(new KeyboardEvent('keyup', { key: 'ArrowLeft' }));
+      expect(el.selectedIndex).to.equal(0);
+    });
+
+    it('stays on current tab if previous or next tab is last tab in that direction and also disabled', async () => {
```

Comment:
If tab 0 is active and we press ArrowLeft we go to the last tab.
So why is the behavior different for disabled tabs?